logger_test.go

```diff
@@ -384,34 +384,54 @@ func TestErrorLogger(t *testing.T) {
 func TestLoggerWithWriterSkippingPaths(t *testing.T) {
 	buffer := new(bytes.Buffer)
 	router := New()
-	router.Use(LoggerWithWriter(buffer, "/skipped"))
+	router.Use(LoggerWithWriter(buffer, "/skipped", "/skippedUsers/:id"))
 	router.GET("/logged", func(c *Context) {})
+	router.GET("/loggedUsers/:id", func(c *Context) {})
 	router.GET("/skipped", func(c *Context) {})
+	router.GET("/skippedUsers/:id", func(c *Context) {})
 
 	PerformRequest(router, "GET", "/logged")
 	assert.Contains(t, buffer.String(), "200")
 
+	buffer.Reset()
+	PerformRequest(router, "GET", "/loggedUsers/2")
+	assert.Contains(t, buffer.String(), "200")
```

Comment:
assert.Contains(t, "200", buffer.String())

Reply:
Thanks for leaving your feedback @appleboy. I just want to confirm this is the change your want, since the current code base adopts the convention of `assert.Contains(t, buffer.String(), "200")`. I will change them all once confirmed. Let me know.

Reply:
@hanyucui sorry for my mistake. you are right.

```go
func Contains(t TestingT, s, contains interface{}, msgAndArgs ...interface{}) bool
```

Reply:
Thanks for double-checking, @appleboy. Let me know if other changes are needed.